<launch>
    <node name = "diff_odom" pkg = "roboteq_motor_controller_driver" type = "diff_odom" output = "screen">
        <rosparam command="load" file="$(find roboteq_motor_controller_driver)/config/odom_conf.yaml" />
    </node>
</launch>
